src/AppBundle/Resources/config/routing.yml

```diff
@@ -1,30 +1,11 @@
-
 app_add_component:
-    path:     /components/add
-    defaults: { _controller: AppBundle:App:addComponent }
+    path:     /components/{dashboardId}/add
+    defaults: { _controller: AppBundle:Component:addComponent }
+    requirements:
+        componentId: \d+
```

Comment:
Yes but here there is no reference to componentId in the URL so I don't understand why do you put this `requirements` ?